docs/docsite/rst/user_guide/playbooks_filters.rst

```diff
@@ -100,6 +100,22 @@ Jinja2 provides a useful 'default' filter that is often a better approach to fai
 In the above example, if the variable 'some_variable' is not defined, the value used will be 5, rather than an error
 being raised.
 
+
+Beginning in version 2.8, attempting to access an attribute of an Undefined value in Jinja will return another Undefined value, rather than throwing an error immediately. This means that you can now simply use
+a default with a value in a nested data structure when you don't know if the intermediate values are defined.
+
+In Ansible 2.8::
```

Comment:
The example was already given above the paragraph you added. I don't know that the added detail of repeating this for 2.8 (and the older example for 2.7) is important in this section. Can you delete that and just leave the 'Beginning in version 2.8...` paragraph in this PR?